app/assets/stylesheets/application.scss

```
@@ -12,3 +12,8 @@ $govuk-images-path: "/assets/images/";
 html * {
   font-family: open-sans, sans-serif !important;
 }
+
+.contact-box {
+  background-color: #f3f2f1; // GOVUK light grey
```

Would it make sense to use the GOV.UK Sass functions for this?

```suggestion
  background-color: govuk-colour("light-grey", $legacy: "grey-3");
```

---

Didn't know that existed, thanks!